packages/ui/package.json

```
@@ -20,7 +20,8 @@
     "type-check": "tsc --pretty --noEmit",
     "type-check:ci": "tsc-absolute --pretty --noEmit",
     "lint:fix": "eslint . --fix",
-    "lint:report": "eslint . --format json --output-file ../../lint-results/ui.json"
+    "lint:report": "eslint . --format json --output-file ../../lint-results/ui.json",
+    "build-icons": "node ./scripts/build-icons.mjs"
```

```suggestion
    "build:icons": "node ./scripts/build-icons.mjs"
```

To match the READMEs and comments